yarn.lock

```diff
@@ -12608,6 +13104,15 @@ wcwidth@^1.0.0:
   dependencies:
     defaults "^1.0.3"
 
+web3-bzz@1.0.0-beta.35:
```

Comment:
Can we trace where does this web3 beta35 come from? I'd like to avoid introducing these dependencies, even if it's for development.

Reply:
These came from our fork of `ganache-core` for coverage runs (https://github.com/OpenZeppelin/ganache-core-coverage), which the beta version of `test-environment` we were using depended on. I updated it to the actual release and these are now gone.